qdrant/v1.1.x/integrations.md

```diff
@@ -6,6 +6,67 @@ Qdrant is a vector database performing an approximate nearest neighbours search
 as a standalone system, yet, in some cases, you may find it easier to implement your semantic search application using some
 higher-level libraries. Some of such projects provide ready-to-go integrations and here is a curated list of them.
 
+## LangChain
+
+LangChain is a library that makes developing Large Language Models based applications much easier. It unifies the interfaces 
+to different libraries, including major embedding providers and Qdrant. Using LangChain, you can focus on the business value 
+instead of writing the boilerplate.
+
+Langchain comes with the Qdrant integration by default. It might be installed with pip:
+
+```bash
+pip install langchain
+```
+
+Qdrant acts as a vector index that may store the embeddings with the documents used to generate them. There are various ways 
+how to us it, but calling `Qdrant.from_texts` is probably the most straightforward way how to get started:
+
+```python
+from langchain.vectorstores import Qdrant
+from langchain.embeddings import HuggingFaceEmbeddings
+
+embeddings = HuggingFaceEmbeddings(
+    model_name="sentence-transformers/all-mpnet-base-v2"
+)
+doc_store = Qdrant.from_texts(
+    texts, embeddings, host=QDRANT_HOST, api_key=QDRANT_API_KEY, collection_name="texts"
+)
+```
```

Comment:
the common question I see is how to use already stored vectors, maybe we can put a small example here?